sidebars.js

```diff
@@ -196,6 +196,7 @@ const sidebars = {
       collapsed: false,
       items: [
         'subnets/reference-cli-commands',
+        'subnets/reference-elastic-subnets-parameters'
```

Comment:
placed here since the `subnets/reference-elastic-subnets-parameters` is intended to be another reference